app/boards/shields/jian/jian_left.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "jian.dtsi"
+
+&kscan0 {
+	col-gpios
+		= <&pro_micro_d  1 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_d  0 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_d  8 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_a  1 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_a  0 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_d 16 GPIO_ACTIVE_HIGH>
+		;
+};
+
+&bt_unpair_combo {
+	key-positions = <0 40>;
+};
```

Comment:
I think this is a deprecated feature. Should probably be removed.

Reply:
It was deprecated an hour ago. Removed all mentions of it.